.travis.yml

```diff
@@ -162,6 +162,9 @@ install:
   # Raylet tests.
   - ./ci/suppress_output bash src/ray/test/run_object_manager_tests.sh
   - ./ci/suppress_output bazel test --build_tests_only --test_lang_filters=cc //:all
+
+  # Streaming cpp test.
+  - ./ci/suppress_output bash streaming/src/test/run_tests_with_streamingqueue.sh
```

Comment:
Can we make sure to use `bazel test` instead of a bash script to run the tests? For example, the tests in core_worker/test like `direct_actor_transport_test.cc` can be run with `bazel test --test_output=streamed //:direct_actor_transport_test` which is much easier to work with that the existing core_worker test script.

Reply:
> Can we make sure to use `bazel test` instead of a bash script to run the tests? For example, the tests in core_worker/test like `direct_actor_transport_test.cc` can be run with `bazel test --test_output=streamed //:direct_actor_transport_test` which is much easier to work with that the existing core_worker test script.

This test starts raylet redis-server, creates two actors to send/receive streaming data through direct actor call, just like `./ci/suppress_output bash src/ray/test/run_core_worker_tests.sh` does. `bazel test` should not work.